satellite/console/service.go

```diff
@@ -27,6 +27,8 @@ const (
 	maxLimit            = 50
 	tokenExpirationTime = 24 * time.Hour
 
+	resetPasswordExpirationTime = 24 * time.Hour
```

Comment:
do we need this variable? tokenExpirationTime will be enough

Reply:
agree